src/foundry/client-esm/canvas/sources/point-sound-source.d.mts

```diff
@@ -0,0 +1,33 @@
+import type BaseEffectSource from "./base-effect-source.d.mts";
+import PointEffectSourceMixin, {
+  type PointEffectSourceMixin_BaseEffectSource_Interface,
+} from "./point-effect-source.mts";
+
+declare const PointEffectSourceMixin_BaseEffectSource: PointEffectSourceMixin_BaseEffectSource_Interface;
+
+type SoundSourceData = PointEffectSourceMixin.PointEffectSourceData & BaseEffectSource.BaseEffectSourceData;
+
+/**
+ * A specialized subclass of the BaseEffectSource which describes a point-based source of sound.
+ */
+export default class PointSoundSource<
+  SourceData extends SoundSourceData = SoundSourceData,
+  SourceShape extends PointSourcePolygon = PointSourcePolygon,
+> extends PointEffectSourceMixin_BaseEffectSource<SourceData, SourceShape> {
+  /** @defaultValue `"sound"` */
+  static override sourceType: string;
+
+  override get effectsCollection(): Collection<this>;
+
+  override _getPolygonConfiguration(): PointSourcePolygonConfig;
+
+  /**
+   * Get the effective volume at which an AmbientSound source should be played for a certain listener.
+   */
+  getVolumeMultiplier(
+    listener: Point,
+    options?: {
+      easing?: boolean;
```

Comment:
probably add `undefined` as an option for `easing` (an object property), This is not necessary for `options` as that's a function parameter.